src/Sylius/Bundle/OrderBundle/Form/Type/OrderItemType.php

```diff
@@ -45,9 +47,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('quantity', 'integer', [
                 'label' => 'sylius.form.order_item.quantity',
+                'constraints' => [
+                    new NotNull(),
+                    new Type(['type' => 'numeric']),
+                ],
             ])
             ->add('unitPrice', 'sylius_money', [
                 'label' => 'sylius.form.order_item.unit_price',
+                'constraints' => [
+                    new NotNull(),
+                    new Type(['type' => 'numeric']),
```

Comment:
Ok, for me it could be left that way, but we should consider shall we keep this ``Assert``s in entities models.